tests/e2e/translations/message-format.spec.ts.js

```diff
@@ -0,0 +1,51 @@
+const utils = require('../../utils');
```

Comment:
Should this file be `spec.js`?

Reply:
yep! thanks for noticing. 
I keep wondering what happened here, I think I added a new .js file and the IDE added the .js extension, but I typed .ts in the name. 🤦🏻